tests/policies/test_exploration_wrapper.py

```diff
@@ -57,11 +57,46 @@ def test_switch():
     assert num_constant > 450
 
 
+def test_switch_prob():
+    # Ensure that we test both the random and the wrapped policy
+    # at least once:
+    wrapper, venv = make_wrapper(random_prob=0.5, switch_prob=0.0)
+    policy = wrapper.current_policy
+    np.random.seed(0)
+    obs = np.random.rand(100, 2)
+    for action in wrapper(obs):
+        assert venv.action_space.contains(action)
+        assert wrapper.current_policy == policy
+
+    for random_prob in [0.0, 0.5]:
+        wrapper, venv = make_wrapper(random_prob=random_prob, switch_prob=1.0)
```

Comment:
I suggest factoring this code block out into a helper function, then calling it via something like:

```
num_random, num_constant = _wrapper_sample(random_prob=0.5, num_timesteps=1000, seed=0)
assert num_random > 450
assert num_constant > 450
num_random, num_constant = _wrapper_sample(random_prob=0.0, num_timesteps=1000, seed=0)
assert num_random == 0
assert num_constant == 1000
```

For loop seems a bit weird when we then have an if-else statement for every value the for loop can take.

Reply:
Sounds good! I made corresponding changes to this.